test/integration/backward_compatible/topic/ReliableTopicOnClusterRestart.js

```diff
@@ -92,6 +96,9 @@ describe('ReliableTopicOnClusterRestartTest', function () {
             messageArrived = true;
         });
 
+        // wait some time for message listener to initialized
+        await promiseWaitMilliseconds(2000);
```

Comment:
I don't like this kind of fixed-duration sleeps because they are prone to failures.

Can we try to do something else, like spy the `ReliableTopicListenerRunner` and wait until the `runner.next()` is called via assertTrueEventually?